### ISSUE TITLE
New tag please

### ISSUE THREAD

**Issue by SaeedDev94** [kelvinmo/simplejwt#182]
Can't update to Laravel 11 because it requires symfony 7

**Comment by kelvinmo** [comment]
Please provide the output of `composer why-not` with Laravel 11 so that we can see why it is blocked.

**Comment by SaeedDev94** [comment]
I closed the console but I fixed the issue by using master branch as a temp solution  
```
{
  "kelvinmo/simplejwt": "dev-master#c137a3a52ae009906c9409a49fffd28c6758d77c"
}
```

**Comment by kelvinmo** [comment]
10b576c will need to be backported to the 0.8 branch to add `^7.0` to the dependency on `symfony/console`.

**Comment by kelvinmo** [comment]
Fixed with [v0.8.2](https://github.com/kelvinmo/simplejwt/releases/tag/v0.8.2)
